### PR TITLE
Update nuspec to support jquery v1.9.1 - v3.x

### PR DESCRIPTION
**Pull request by supergibbs**
@DavidDeSloovere emailed about the latest NuGet package requiring jquery v3.0.0.1, this will update the nuspec to match the bower.json file and require jquery to be at least v1.9.1 and less than v4.0.0

Also updates 
- the version to plain 4.0.0 (it's set on publishing to match the package.json file)
- the `tags` to match `keywords` of package.json

### CODE REVIEW

File: nuget/bootstrap.sass.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>bootstrap.sass</id>
-    <version>4.0.0-alpha.3</version>
```
Comment (bardiharborow) [inline nuget/bootstrap.sass.nuspec]: Why did you remove the `-alpha.3` rather than updating it?

Comment (supergibbs) [inline nuget/bootstrap.sass.nuspec]: less churn, the nuget publishing script sets the version to match the project.json file so now there is one less thing to update for version bumps.

**Review by bardiharborow — APPROVED** [review]

File: nuget/bootstrap.nuspec [hunk 2]
```diff
@@ -16,9 +16,9 @@
     <copyright>Copyright 2017</copyright>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
-      <dependency id="jQuery" version="3.0.0.1" />
+      <dependency id="jQuery" version="[1.9.1,3)" />
```
Comment (tillkruss) [inline nuget/bootstrap.nuspec]: Is this correct? Square opening bracket and round closing parenthesis?

Comment (fchiumeo) [inline nuget/bootstrap.nuspec]: in [nuget packages](https://docs.nuget.org/ndocs/create-packages/dependency-versions#version-ranges) yes

Comment (supergibbs) [inline nuget/bootstrap.nuspec]: Yes, but I did mess up values. Just updated to `[1.9.1,4)` which means greater or equal to v1.9.1 but less than v4

**Comment by supergibbs** [comment]
Squashed 3 commits into one final one
